File: sdks/php/src/Taxes/Requests/ListTaxesRequest.php
<?php

namespace NovaBilling\Taxes\Requests;

use NovaBilling\Core\Json\JsonSerializableType;

class ListTaxesRequest extends JsonSerializableType
{
    /**
     * @var ?bool $appliedByDefault
     */
    public ?bool $appliedByDefault;

    /**
     * @var ?float $page
     */
    public ?float $page;

    /**
     * @var ?float $limit
     */
    public ?float $limit;

    /**
     * @param array{
     *   appliedByDefault?: ?bool,
     *   page?: ?float,
     *   limit?: ?float,
     * } $values
     */
    public function __construct(
        array $values = [],
    ) {
        $this->appliedByDefault = $values['appliedByDefault'] ?? null;
        $this->page = $values['page'] ?? null;
        $this->limit = $values['limit'] ?? null;
    }
}

File: sdks/csharp/src/NovaBilling/Taxes/Requests/ListTaxesRequest.cs
using System.Text.Json.Serialization;
using NovaBilling.Core;

namespace NovaBilling;

[Serializable]
public record ListTaxesRequest
{
    [JsonIgnore]
    public bool? AppliedByDefault { get; set; }

    [JsonIgnore]
    public double? Page { get; set; }

    [JsonIgnore]
    public double? Limit { get; set; }

    /// <inheritdoc />
    public override string ToString()
    {
        return JsonUtils.Serialize(this);
    }
}
